rotkehlchen/chain/ethereum/interfaces/ammswap/types.py

```diff
@@ -72,6 +89,17 @@ class ProtocolBalance(NamedTuple):
     unknown_assets: Set[EthereumToken]
 
 
+class V3ProtocolBalance(NamedTuple):
+    """Container structure for uniswap V3 LP balances
+
+    Known assets are all assets we have an oracle for
+    Unknown assets are those we would have to try to query through uniswap directly
+    """
+    address_balances: AddressToV3LPBalances
+    known_assets: Set[EthereumToken]
+    unknown_assets: Set[EthereumToken]
```

Comment:
Why creating a new structure for this? Can't this one inherit from `ProtocolBalance` or just use `ProtocolBalance`?

Reply:
Sorry, subclassing `ProtocolBalance` and changing the fields raises an error because it's a named tuple